modules/admin-ui-picard/app/src/components/events/partials/wizards/NewEventWizard.js

```diff
@@ -113,60 +112,68 @@ const NewEventWizard = ({ metadataFields, close, postNewEvent, uploadAssetOption
                         validationSchema={currentValidationSchema}
                         onSubmit={values => handleSubmit(values)}>
                     {/* Render wizard pages depending on current value of page variable */}
-                    {formik => (
-                      <>
-                          {/* Stepper that shows each step of wizard as header */}
-                          <WizardStepperEvent steps={steps}
-                                              page={page}
-                                              setPage={setPage}
-                                              completed={pageCompleted}
-                                              setCompleted={setPageCompleted}
-                                              formik={formik}/>
-                          <div>
-                              {page === 0 && (
-                                <NewMetadataPage nextPage={nextPage}
-                                                 formik={formik}
-                                                 metadataFields={metadataFields}
-                                                 header={steps[page].translation} />
-                              )}
-                              {page === 1 && (
-                                // todo: finish implementation when information about endpoints and structure are gathered
-                                <NewMetadataExtendedPage previousPage={previousPage}
-                                                         nextPage={nextPage}
-                                                         formik={formik}
-                                                         header={steps[page].translation} />
-                              )}
-                              {page === 2 && (
-                                <NewSourcePage previousPage={previousPage}
-                                               nextPage={nextPage}
-                                               formik={formik} />
-                              )}
-                              {page === 3  && (
-                                <NewAssetUploadPage previousPage={previousPage}
-                                                    nextPage={nextPage}
-                                                    formik={formik} />
-                              )}
-                              {page === 4 && (
-                                <NewProcessingPage previousPage={previousPage}
+                    {formik => {
+
+                        // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
this comment is a hint for ESlint to let the violation of using `useEffect` at this position slip. If you would remove this comment, an error is shown because of the `useEffect` at this position.

Reply:
maybe put a comment above this comment line, explaining that